### ISSUE TITLE
IOS -  Bottom of Preference page is not transparent

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#27535]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Action Performed:
1. Login with any account on iPhone X and above
2. Go Settings => Preferences
3. Scroll up and down, observer the bottom of the page.

## Expected Result:
Bottom of the Preference page should be transparent

## Actual Result:
Bottom of the Preference page is not transparent

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.70.5
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/8e806bd0-14fc-499b-badd-4c7d53d1225f

![IMG_8650 (3)](https://github.com/Expensify/App/assets/93399543/bf7291e3-71f3-4cd9-8781-71a63e83b5e1)

![IMG_B0CFC2EBAFBD-1 (1)](https://github.com/Expensify/App/assets/93399543/35abdf02-16ae-4387-bafb-f5cafd7f5bde)

**Expensify/Expensify Issue URL:**
**Issue reported by:** @hungvu193
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1694676966334599

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kbecciv** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Bottom of Preference page is not transparent
### What is the root cause of that problem?
We're adding bottomPadding here which make the bottom isn't transparent with the case we didn't have footer component.
https://github.com/Expensify/App/blob/d68263ad664d8a8a81ca859147696df8abfcd480/src/components/IllustratedHeaderPageLayout.js#L59
### What changes do you think we should make in order to solve the problem?
#### Solution 1 
We only need the padding if we have the footer so we should update the condition:
 ` !isOffline ? safeAreaPaddingBottomStyle : {}`
 to 
`!isOffline && !_.isNull(footer) ? safeAreaPaddingBottomStyle : {}`

#### Solution 2:
Remove ` !isOffline ? safeAreaPaddingBottomStyle : {}` and update style of the `FixedFooter`:
```javascript
const safeAreaMarginBottom = StyleUtils.getSafeAreaMargins(insets);

{!_.isNull(footer) && <FixedFooter style={[!isOffline ? safeAreaMarginBottom : {}]}>{footer}</FixedFooter>}
```
### What alternative solutions did you explore? (Optional)
N/A

**Comment by PiyushChandra17** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Bottom of the Preference page is not transparent

### What is the root cause of that problem?
One condition is missing and it should be the first to check in order `safeAreaPaddingBottomStyle` to work and bottom of the preference page to be transparent ie. we did not have the footer component

https://github.com/Expensify/App/blob/2427a06ed24f0649e55c14dbf4ae1a2a9ebe2109/src/components/IllustratedHeaderPageLayout.js#L62

### What changes do you think we should make in order to solve the problem?
First thing first we need check if we have footer:

`!_.isNull(footer) && !isOffline ? safeAreaPaddingBottomStyle : {}`

### What alternative solutions did you explore? (Optional)
NA

###Result:
![Simulator Screenshot - iPhone 13 - 2023-09-15 at 23 02 44](https://github.com/Expensify/App/assets/47579287/bfe5a843-3352-4970-86d4-ef32f17e23e3)

**Comment by robertKozik** [comment]
Hi @hungvu193 @PiyushChandra17! Thank you both for your proposals. After careful consideration, it's evident that both of you have accurately identified the root cause of the issue, and your proposed solutions are quite similar. Due to the timing, I'm inclined to go with @hungvu193's proposal as it was the first one. Regarding the solution, I'm leaning towards the first option as it seems more descriptive, but I'm open to further discussion on this.

🎀 👀 🎀 C+ reviewed

**Comment by tylerkaraszewski** [comment]
Does anyone know why github won't let me assign @hungvu193? He just will not appear in the dropdown for assignees.

**Comment by laurenreidexpensify** [comment]
Bumped @hungvu193 in Slack too :)

**Comment by PiyushChandra17** [comment]
@robertKozik Sure, we need to check footer first and it's different from what @hungvu193 proposed, it's the vital check first going forward with this one.

Also @laurenreidexpensify if he is unavailable, i can take this going forward :)

**Comment by laurenreidexpensify** [comment]
Payment Summary:  

- External issue reporter @hungvu193 $50 payment issued in Upwork 
-  Contributor that fixed the issue @hungvu193  @500 payment issued in Upwork 
-  Contributor+ that helped on the issue and/or PR @robertKozik agency worker, no payment issued

**Comment by laurenreidexpensify** [comment]
@robertKozik please advise on regression steps in checklist above, thanks

**Comment by robertKozik** [comment]
> * [X]  [@robertKozik] The PR that introduced the bug has been identified. Link to the PR: **Cannot trace any particular PR which introduced this behaviour**
> * [X]  [@robertKozik] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: **N/A**
> * [X]  [@robertKozik] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: **N/A**
> * [X]  [@robertKozik] Determine if we should create a regression test for this bug. **I don't think the regression test is needed here. It's visual bug which don't interfere with any user flow**
> * [X]  [@robertKozik] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again. **N/A**

tldr; I don't think the regression test is needed here. It's visual bug which don't interfere with any user flow
